Docs: Added back v1 resource information (#127169)
grafana/grafana: docs/sources/visualizations/dashboards/build-dashboards/view-dashboard-json-model/index.md

There are currently three dashboard JSON schema models:

- [V2 Resource](#v2-resource-model): The current schema, supporting new features such as advanced layouts and conditional rendering. It models all dashboard elements as Kubernetes kinds, following Kubernetes conventions for declaring dashboard components.

> - [V1 Resource](#v1-resource-model): The Classic dashboard schema formatted as a Kubernetes-style resource. Its `spec` property contains the Classic model of the schema. It was the default format for API communication between Grafana v12.2.0 and v13.0.0 and has been used for exporting, importing, and sharing dashboards.

Reviewer: if this is the default between 12.2 and 13, and classic was the default before 13, it sounds like there's a slight mix up. wouldn't classic be before 12.2 at which point this becomes this default until 13?

> - [Classic](#classic-model): A non-Kubernetes resource that was the default before Grafana v13.0. It's been widely used for exporting, importing, and sharing dashboards in the Grafana dashboards collection at [grafana.com/dashboards](https://grafana.com/grafana/dashboards/). Dashboards created using this model can be exported using either this model or V2.

Author: ```suggestion
- [Classic](#classic-model): A non-Kubernetes resource that was the default before Grafana v12.2. It's been widely used for exporting, importing, and sharing dashboards in the Grafana dashboards collection at [grafana.com/dashboards](https://grafana.com/grafana/dashboards/). Dashboards created using this model can be exported using either this model or V2.
```

There are currently three dashboard JSON schema models:

- [V2 Resource](#v2-resource-model): The current schema, supporting new features such as advanced layouts and conditional rendering. It models all dashboard elements as Kubernetes kinds, following Kubernetes conventions for declaring dashboard components.
- [V1 Resource](#v1-resource-model): The Classic dashboard schema formatted as a Kubernetes-style resource. Its `spec` property contains the Classic model of the schema. It was the default format for API communication between Grafana v12.2.0 and v13.0.0 and has been used for exporting, importing, and sharing dashboards.

> - [Classic](#classic-model): A non-Kubernetes resource that was the default before Grafana v12.2. It's been widely used for exporting, importing, and sharing dashboards in the Grafana dashboards collection at [grafana.com/dashboards](https://grafana.com/grafana/dashboards/). Dashboards created using this model can be exported using either this model or V2.

Author: ```suggestion
- [Classic](#classic-model): A non-Kubernetes resource that was the default before Grafana v12.2.0. It's been widely used for exporting, importing, and sharing dashboards in the Grafana dashboards collection at [grafana.com/dashboards](https://grafana.com/grafana/dashboards/). Dashboards created using this model can be exported using either this model or V2.
```

For the detailed V2 Resource model schema, refer to the [Swagger documentation](https://play.grafana.org/swagger?api=dashboard.grafana.app-v2beta1).
You can also reach the Swagger schema documentation from your Grafana Cloud instance at: https://grafana.com/launch/swagger.

## V1 Resource model

The V1 Resource schema model formats the [Classic JSON model](#classic-model) schema as a Kubernetes-style resource.
The `spec` property of the schema contains the Classic-style model of the schema.

> For the detailed V1 Resource model schema, refer to the [Swagger documentation](https://play.grafana.org/swagger?api=dashboard.grafana.app-v1beta1).

Reviewer 2: ```suggestion
For the detailed V1 Resource model schema, refer to the [Swagger documentation](https://play.grafana.org/swagger?api=dashboard.grafana.app-v1).
```

You can also reach the Swagger schema documentation from your Grafana Cloud instance at: https://grafana.com/launch/swagger.

The following code snippet shows the fields included in the V1 Resource model:

```json
{

>   "apiVersion": "dashboard.grafana.app/v1beta1",

Reviewer 2: ```suggestion
  "apiVersion": "dashboard.grafana.app/v1",
```
